### PR TITLE
Javascript data model

### PR DESCRIPTION
**Pull request by DreSean-Sutton**
image
image
image
image

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    shipTo: 'JS Masher',
+    orderPlaced: 'August 5, 2020',
+    deliveryDate: 'July 7, 2020',
+    orderNumber: '114-2875557-9059409',
+    returnWindow: 'August 5, 2020',
+    total: 17.22,
+    items: [
+      {
+        title: 'Gamecube Controller Adapter',
+        author: NaN,
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js]: `NaN`? How about `null`?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-data-model/order-history.js (hunk 1 above)
Comment (DreSean-Sutton) [reply javascript-data-model/order-history.js]: Forgot to change that to something better later lmao

**Review by codingcodymiller — APPROVED** [review]

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
WAIT

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
You still have other instances of NaN in your code, please always make sure to look if feedback needs to be addressed in multiple places in your code 👀

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**